dist.ini

```diff
@@ -25,9 +25,7 @@ remove = strict
 remove = warnings
 remove = constant
 remove = Exporter
-remove = Getopt::Std
```

Comment:
This was needed by `pwhich` which was spun off into `App::pwhich` quite some time ago.